clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/provider/agent/TaskHealthCachingAgent.java

```diff
@@ -85,10 +90,11 @@ public TaskHealthCachingAgent(String accountName, String region, AmazonClientPro
 
   @Override
   protected List<TaskHealth> getItems(AmazonECS ecs, ProviderCache providerCache) {
+    TaskCacheClient taskCacheClient = new TaskCacheClient(providerCache, objectMapper);
```

Comment:
I dont think this should be instantiated each time, since it doesn't require a regionalized/account-specific ECS client

Reply:
There'd be circular dependency otherwise.  It's "OK" to have it like that then